tests/test_routing.py

```diff
@@ -443,7 +443,11 @@ def test_url_for_with_root_path():
 
 
 double_mount_routes = [
-    Mount("/mount", name="mount", routes=[Mount("/static", ..., name="static")]),
+    Mount(
+        "/mount",
+        name="mount",
+        routes=[Mount("/static", ..., name="static")],  # type: ignore[arg-type]
```

Comment:
Example error without this ignore:
```
tests/test_routing.py:449: error: Argument 2 to "Mount" has incompatible type "ellipsis"; expected "Optional[Callable[[MutableMapping[str, Any], Callable[[], Awaitable[MutableMapping[str, Any]]], Callable[[MutableMapping[str, Any]], Awaitable[None]]], Awaitable[None]]]"
Found 1 error in 1 file (checked 19 source files)
```

Reply:
Should we pass a stub `app(scope, receive, send)` instead?